src/global/coin-type.ts

```diff
@@ -322,7 +326,8 @@ export class CoinType {
     name            : CoinName[CoinName.MatchPool],
     addressFormat   : ETHEREUM_ADDRESS_FORMAT,
     dust            : 1,
-    defaultDecimals : 3
+    defaultDecimals : 3,
+    exchangeForbidden: true,
```

Comment:
Recommend adding a comment to say why, since the source will be public here pretty soon. Perhaps:

```
exchangeForbidden: true, // Temporarily disabled because of a bug in the expected withdrawal amount calculation.
```